.gitignore

```diff
@@ -1,13 +1,18 @@
+bookenv/
 /build
+.coverage
+coverage.xml
+dist/
 /env*
+env*/
 *.egg
 *.egg-info
+.eggs/
```

Comment:
Not a huge fan of cargo culting this and many of the other entries into the `.gitignore` I assume this too came from pyramid?

Reply:
Yes and based on @stevepiercy comments, the final .gitignore is around half the size of pyramid's one.